sorting/quicksort3.cpp

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
```

Comment:
Please include a good documentation for your program. 
You can refer #845

Reply:
okay